File: src/main/java/com/ruolan/weatherserver/WeatherserverApplication.java
package com.ruolan.weatherserver;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class WeatherserverApplication {

	public static void main(String[] args) {
		System.out.println("启动了..........");
		SpringApplication.run(WeatherserverApplication.class, args);
	}
}

File: src/main/java/com/ruolan/weatherserver/WeatherserverApplication.kt
package com.ruolan.weatherserver

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
open class WeatherserverApplication {

    fun main(args: Array<String>) {
        println("启动了..........")
        SpringApplication.run(WeatherserverApplication::class.java, *args)
    }

}
